platform/config/podspec.yaml

```diff
@@ -4,19 +4,6 @@ home: /content/amp-dev/index.html
 localization:
   root_path: /{locale}/
   default_locale: en
```

Comment:
Did you intentionally remove all languages?

Reply:
As per an earlier review: https://github.com/ampproject/docs/pull/1881/files#r275148312 😁 You correctly made the point that they are duplicated by https://github.com/ampproject/docs/blob/8daff99905fe8af1463e7a4e6263323aeaa6ff59/platform/lib/config.js#L30

Reply:
hah! I thought pod spec.yaml would be the single source of truth, but the other way around is fine as well.